tests/integration/test_api/conftest.py

```diff
@@ -73,17 +73,17 @@ def configure_api_environment(get_configuration, request):
 
     if hasattr(request.module, 'force_restart_after_restoring'):
         if getattr(request.module, 'force_restart_after_restoring'):
-            subprocess.call([os.path.join(WAZUH_PATH, 'bin', 'wazuh-apid'), 'restart'])
+            subprocess.call([os.path.join(WAZUH_PATH, 'bin', 'ossec-control'), 'restart'])
 
 
 @pytest.fixture(scope='module')
 def restart_api(get_configuration, request):
```

Comment:
It is not possible to use the restart_wazuh instead of this as those fixtures works in the same way but updates different log files. As they are fixtures we cannot simply set the log filename as an argument for the function.